.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
 .drone.sec.yml
+.idea/
+debug.test
```

Comment:
I remember these were already added in the markdown parser tests?

Reply:
Yeah was working in parallel so had to kind of add this in each.  Should come out in the merge.